### PR TITLE
Added script for reading esc data

### PR DESCRIPTION
**Pull request by EugeneShilin**

### CODE REVIEW

File: drivers/esc-align700/esc.cpp [hunk 1]
```diff
@@ -0,0 +1,163 @@
+#include <apx.h>
+
+#pragma pack(1) //needed for correct work of memcpy() from serial to struct
```
Comment (uavinda) [inline drivers/esc-align700/esc.cpp:3]: i think it's better to use `__attribute packed` like think when the struct is declared.

Comment (EugeneShilin) [reply drivers/esc-align700/esc.cpp:3]: What if we would like to pack both msg1 and msg2 structures, even though we don't use msg2 yet? Will it still be better to use __attribute packed separately for each of them?

Comment (uavinda) [reply drivers/esc-align700/esc.cpp:3]: I think so, because it means `explicit` packing of a structure.

Comment (uavinda) [reply drivers/esc-align700/esc.cpp:3]: Though you may use pack push/pop behavior over each struct definition otherwise.

Comment (uavinda) [reply drivers/esc-align700/esc.cpp:3]: Alternatively, use

```
#pragma pack(1)

typedef struct1....
typedef struct2....

#pragma pack()
```

File: drivers/esc-align700/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# APX script for align700 esc
+
+Script reads and parses telemetry data from KONTRONIK electronic speed control (ESC) on helicopter Align700
```
Comment (uavinda) [inline drivers/esc-align700/README.md:3]: a link to the `kontronik` hardware would be helpful

**Comment by uavinda** [comment]
The `md` syntax is preferred to comply [`github markdown`](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

**Review by uavinda — APPROVED** [review]
looks ok now

**Pull request merged**